libretranslate/init.py

```diff
@@ -44,10 +47,24 @@ def check_and_install_models(force=False, load_only_lang_codes=None):
 
         # Download and install all available packages
         for available_package in available_packages:
-            print(
-                f"Downloading {available_package} ({available_package.package_version}) ..."
-            )
-            available_package.install()
+            update = False
+            if not force:
+                for pack in installed_packages:
+                    if (
+                            pack.from_code == available_package.from_code
+                            and pack.to_code == available_package.to_code
+                        ):
+                        update = True
+                        if pack.package_version < available_package.package_version:
```

Comment:
Semantic versioning cannot be compared by string comparison:

e.g. "12.0" < "2.9" --> True (should be False)